dsn_config:
  # Sizes
  feature_dim : 64  # 32 would be normal

  # Mean Shift parameters (for 3D voting)
  max_GMS_iters : 10 
  epsilon : 0.05  # Connected Components parameter
  sigma : 0.02 # Gaussian bandwidth parameter
  num_seeds : 200 # Used for MeanShift, but not BlurringMeanShift
  subsample_factor : 5
  
  # Misc
  min_pixels_thresh : 500
  tau : 15.0

rrn_config:
  # Sizes
  feature_dim : 64 # 32 would be normal
  img_H : 224
  img_W : 224
  
  # architecture parameters
  use_coordconv : False

uois3d_config:
  # Padding for RGB Refinement Network
  padding_percentage : 0.25
  
  # Open/Close Morphology for IMP (Initial Mask Processing) module
  use_open_close_morphology : True
  open_close_morphology_ksize : 9
  
  # Largest Connected Component for IMP module
  use_largest_connected_component : True
